### PR TITLE
Fix API issue in generate_headline function

### PR DESCRIPTION
**Pull request by lgaleana-llm**
This PR fixes the issue in the generate_headline function where the response from the OpenAI API was being accessed incorrectly. The response structure has been corrected, and the corresponding unit tests have been updated.

### Test Plan

pytest / playwright

### CODE REVIEW

**Comment by lgaleana-llm** [comment]
Time to create PR: 72.88 seconds
OPenAI cost: 0.30238499999999996

File: app/openai_utils.py [hunk 1]
```diff
@@ -12,4 +12,4 @@ def generate_headline(client, text: str, image_url: str) -> str:
         ],
         max_tokens=50
     )
-    return response.choices[0].message['content'].strip()
+    return response.choices[0].message.content.strip()  # Corrected this line
```
Comment (lgaleana) [inline app/openai_utils.py]: Remove this comment

Comment (lgaleana-llm) [reply app/openai_utils.py:15]: Sure, I'll remove the comment. Thank you for the feedback!

**Comment by lgaleana-llm** [comment]
Time to amend PR: 2.52 secondsOPenAI cost: 0.09473

File: app/openai_utils.py (hunk 1 above)
Comment (lgaleana-llm) [reply app/openai_utils.py:15]: Comment removed.

**Comment by lgaleana-llm** [comment]
Time to amend PR: 2.51 secondsOPenAI cost: 0.09451000000000001

File: app/openai_utils.py (hunk 1 above)
Comment (lgaleana-llm) [reply app/openai_utils.py:15]: Comment removed.

**Comment by lgaleana-llm** [comment]
Time to amend PR: 2.68 secondsOPenAI cost: 0.09451000000000001

File: app/openai_utils.py (hunk 1 above)
Comment (lgaleana-llm) [reply app/openai_utils.py]: Sure, I'll remove the comment from line 15 in the `app/openai_utils.py` file.

### Updated `app/openai_utils.py`

```python
from openai import OpenAI
import os

def generate_headline(client, text: str, image_url: str) -> str:
    prompt = f"Generate an ad headline for the following image URL based on the website text:\n\nWebsite Text:\n{text}\n\nImage URL: {image_url}\n\nAd Headline:"
    response = client.chat.completions.create(
        model="gpt-3.5-turbo",
        messages=[
            {"role": "system", "content": "You are a helpful assistant."},
            {"role": "user", "content": prompt}
        ],
        max_tokens=50
    )
    return response.choices[0].message.content.strip()
```

I'll go ahead and make this change.

**Comment by lgaleana-llm** [comment]
Time to amend PR: 25.45 secondsOPenAI cost: 0.361285

**Review by lgaleana — APPROVED** [review]

**Pull request merged**